### ISSUE TITLE
Corruption of data with quotes when exporting via csv Export

### ISSUE THREAD

**Issue by mmlitvin** [primefaces/primeng#4615]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
http://plnkr.co/edit/65BurX0PYEC99LGtPkNH?p=preview

**Current behavior**
Data containing double quotes is corrupted during export

**Expected behavior**
True data preservation

**Minimal reproduction of the problem with instructions**
See plunker

* **PrimeNG version:** 
4.3.0

* **Browser:**
Chrome

**Comment by cagataycivici** [comment]
Should be fixed with 5.0.2.

**Comment by waratah** [comment]
see also #11618
